app/schemas/schemas_campaign.py

```diff
@@ -109,3 +109,13 @@ class Config:
 class VoteStats(BaseModel):
     section_id: str
     count: int
+
+
+class Voters(BaseModel):
+    """Base schema for a vote."""
+
+    id_voters: str
+    group: str
+
+    class Config:
+        orm_mode = True
```

Comment:
Could you add a docstring explaining the purpose of this schema?

Reply:
If I understood correctly, the table contains identifiers of groups that are allowed to vote

Reply:
Maybe we should find a more explicit name